app/Notifications/User/Conversation.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Notifications\User;
+
+use App\Channels\ConversationChannel;
+use App\Models\Message;
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class Conversation extends Notification
```

Comment:
this class name is wrong - this is not a new conversation, its a message

Reply:
done